gists/frontend-pull-request-guidelines.md

```diff
@@ -1,6 +1,23 @@
 # Frontend Pull Request Guidelines
 This document details guidelines for reviewing frontend (especially LPL React-flavored) code. Some of these suggestions may apply more broadly to software development in general, but are mentioned here specifically based on the frequency with which they have been observed in client-side code.
 
+## General
+- Strive for consistency. Review other areas of the code for patterns, copy and paste, make things easier for yourself.
```

Comment:
"Make things easier for yourself" sounds like an additional instruction, but it's not very clear. Is the sentiment that you're trying to articulate that reviewing code and copying and pasting will make things easier on yourself? Or is this a general philosophy that you think all PR authors should follow?